app/src/main/java/org/schabi/newpipe/util/DeviceUtils.java

```diff
@@ -35,22 +35,6 @@ public final class DeviceUtils {
     private static Boolean isTV = null;
     private static Boolean isFireTV = null;
 
-    /*
-     * Devices that do not support media tunneling
-     */
-    // Formuler Z8 Pro, Z8, CC, Z Alpha, Z+ Neo
-    private static final boolean HI3798MV200 = Build.VERSION.SDK_INT == 24
-            && Build.DEVICE.equals("Hi3798MV200");
-    // Zephir TS43UHD-2
-    private static final boolean CVT_MT5886_EU_1G = Build.VERSION.SDK_INT == 24
-            && Build.DEVICE.equals("cvt_mt5886_eu_1g");
-    // Hilife TV
-    private static final boolean REALTEKATV = Build.VERSION.SDK_INT == 25
-            && Build.DEVICE.equals("RealtekATV");
-    // Philips QM16XE
-    private static final boolean QM16XE_U = Build.VERSION.SDK_INT == 23
-            && Build.DEVICE.equals("QM16XE_U");
```

Comment:
Mmmh, why are you removing this list completely? We know all these TVs don't work, so by default on such devices tunneling should be disabled. Then the user can go into settings and enable it, but at least it should be off by default.

Reply:
Someone needs to write a settings migration for that. Otherwise, NewPipe is going to stop working for some users and they certainly not search for a solution in the settings.